src/main/java/io/quarkus/search/app/hibernate/InputProviderHtmlBodyTextBridge.java

```diff
@@ -18,7 +18,7 @@ public String toIndexedValue(InputProvider provider, ValueBridgeToIndexedValueCo
         try (var in = provider.open()) {
             Element body = Jsoup.parse(in, StandardCharsets.UTF_8.name(), "/").body();
             // Content div has two grid columns: actual content and TOC. There's not much use of the TOC, we want the content only:
-            Element content = body.selectFirst(".content .grid__item");
+            Element content = body.selectFirst(".content .grid_item");
```

Comment:
I don't think that's right? At least it causes many problems when indexing:

```
2023-11-22 09:02:12,305 WARN  [io.qua.sea.app.hib.InputProviderHtmlBodyTextBridge] (executor-thread-1) Was unable to find the content section of a guide. Using whole document as text. GitInputProvider{path='version/2.13/guides/all-builditems.html'}
```

If I revert this change, the warnings disappear.

Reply:
ah I think this is IntelliJ refactor good catch!!!!!!